### DISCUSSION TITLE
Why model.related('x').createMany doesn't attach relations to model

### DISCUSSION THREAD

**Discussion by brkn** [adonisjs/core#3879] [General]
Let's say I create a parent record and related children records inside a transaction. After the transaction commits I expect to be able to reach children by calling `parent.children`. Why do we need to call `await parent.load('children')` as another db query?

Example:
```ts
const book = await Database.transaction(async (trx) => {
  const book = await Book.create(
    {
      name: 'Adonis',
    },
    { client: trx }
  )
  // const pingSequence = await PingSequence.query

  await book.related('chapters').createMany(
    chapters.map((chapter, index) => ({
      name: chapter.title,
      chapterNum: index,
    })),
    { client: trx }
  )

  return book
})

console.log('preloaded relations', book.$preloaded)
console.log('sideloaded relations', book.$sideloaded)
```

**Comment by thetutlage** [comment]
Yup that is an improvement we have to bring to the ORM. Right now calling `load` is the only way to get the relationships.
